src/experiments.js

```diff
@@ -73,7 +76,21 @@ export function isDevChannelVersionDoNotUse_(win) {
  * @return {boolean}
  */
 export function isExperimentOn(win, experimentId) {
-  return getExperimentIds(win).indexOf(experimentId) != -1;
+  if (experimentId in EXPERIMENT_TOGGLES) {
+    return EXPERIMENT_TOGGLES[experimentId];
+  }
+
+  const cookieFlag = getExperimentIds(win).indexOf(experimentId) != -1;
```

Comment:
I think it'd be a bit better as:

```
return EXPERIMENT_TOGGLES[experimentId] = calcExperimentOn(win, experimentId);
```

Otherwise, repeated assign+return seems a bit error-prone and duplicative.